apps/src/templates/teacherHomepage/ManageSectionsCollapsible.jsx

```diff
@@ -18,7 +18,7 @@ const ManageSectionsCollapsible = React.createClass({
         linkText={i18n.editSections()}
         link="https://code.org/teacher-dashboard#/sections"
       >
-      {sections ? (
+      {sections.length > 0 ? (
```

Comment:
Does it? The SectionsTable component is just a table. Why put the table in a div?